src/Drivers/DiscordNotificationDriver.php

```diff
@@ -24,31 +26,41 @@ public static function getSettingsView(): string
      */
     public static function getRegistrationView(): string
     {
+
         return 'seatnotifications::discord.registration';
     }
 
     /**
      * @return string
      */
-    public static function getButtonLabel() : string
+    public static function getButtonLabel(): string
     {
+
         return 'Discord';
     }
 
     /**
      * @return string
      */
-    public static function getButtonIconClass() : string
+    public static function getButtonIconClass(): string
     {
+
         return 'fa-bullhorn';
     }
 
+    public static function getPrivateRegistrationRoute() : string
```

Comment:
👍

It has to be in the interface though

Reply:
You mean as a `public static function` ? 
the private registration route must be implemented per driver right?

Reply:
Indead, but the purpose of an interface is to declare a contract which have to be meet by its implementing classes.